packages/styles/src/navigation.scss

```diff
@@ -378,7 +400,7 @@ $navListContainerSubmenu: #{$block}__list-container--submenu;
       }
     }
 
-    .#{$navItemWithExpander} {
+    @include _fd-vertical-nav-with-expander() {
```

Comment:
how is this a better approach?

Reply:
@InnaAtanasova previous line would apply styling for both horizontal and vertical navs. This mixin allows to apply styling only for a vertical nav. Same with `_fd-horizontal-nav-with-expander` mixin.